src/mlpack/tests/krann_search_test.cpp

```diff
@@ -274,6 +274,7 @@ BOOST_AUTO_TEST_CASE(SingleDatasetSearch)
 // Test single-tree rank-approximate search with cover trees.
 BOOST_AUTO_TEST_CASE(SingleCoverTreeTest)
 {
+  math::RandomSeed(std::time(NULL));
```

Comment:
Looks like you forgot to remove the line here, the same applies to ``DualCoverTreeTest`.